### PR TITLE
Add Marwan to website

### PR DESCRIPTION
**Pull request by marwanali55**
Added my description and images.

### CODE REVIEW

File: about/people.yml [hunk 1]
```diff
@@ -457,4 +457,13 @@ data:
...
     image: sanjana_main.jpg
+  - name: Marwan Ali
+    type: Part Time
+    team: Advanced Research Computing
+    subteam: Computational Biology Core
+    title: Data Science Intern
+    github_username: marwanali55
+    brown_directory_uuid: 80e72ea3-bce9-4685-bc2c-3041b55facfc
+    bio: Marwan is a summer intern working with the Computational Biology Core at CCV. He is an undergraduate student studying Applied Mathematics. He loves reading, playing piano, and writing stand-up comedy.
+    image: marwan_main.PNG
```
Comment (eldu) [inline about/people.yml:468]: This says the image will be a png file, but the images are jpgs. Take a look at the preview link (in a comment on the PR) and try to debug why your profile picture isn't showing up!

Comment (marwanali55) [reply about/people.yml]: Hi Ellen! Sorry about that; I just updated it to marwan_main.jpg. Please let me know if there are any other issues.

File: images/people/marwan_hover.JPG
Comment (galenwinsor) [inline images/people/marwan_hover.JPG:1]: Just one last thing: you need to make the `jpg` in this file lowercase to match `marwan_main`. Love the photo!

**Review by galenwinsor — CHANGES_REQUESTED** [review]

File: images/people/marwan_hover.JPG
Comment (marwanali55) [reply images/people/marwan_hover.JPG:1]: Thanks for letting me know Galen, I just made the change. Please reach out if anything else is needed.

**Review by eldu — APPROVED** [review]
Looks good! Looks like you need to update this branch and you should be able to merge it into main! 

Great job!

**Review by galenwinsor — APPROVED** [review]

**Comment by galenwinsor** [comment]
@marwanali55 I went ahead and fixed the merge conflicts; once the checks run and pass, feel free to merge!

**Pull request merged**